Index: src/main/java/it/polimi/ingsw/project/client/gui/FaithMapGUI.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/it/polimi/ingsw/project/client/gui/FaithMapGUI.java b/src/main/java/it/polimi/ingsw/project/client/gui/FaithMapGUI.java
new file mode 100644
--- /dev/null	
+++ b/src/main/java/it/polimi/ingsw/project/client/gui/FaithMapGUI.java	
@@ -0,0 +1,62 @@
+package it.polimi.ingsw.project.client.gui;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FaithMapGUI extends JInternalFrame {
+    private final List<JButton> tiles;
+
+    public FaithMapGUI() {
+        this.setTitle("FaithMap");
+        this.setVisible(true);
+        this.setLayout(new GridLayout(2, 24));
+        this.tiles = new ArrayList<>();
+        for (int i = 0; i < 23; i++) {
+            this.tiles.add(new JButton());
+        }
+        List<JButton> secondRowForPapalCouncils = new ArrayList<>();
+        for (int i = 0; i < 23; i++) {
+            JButton button = new JButton();
+            if (i != 5 && i != 13 && i != 20) {
+                button.setVisible(false);
+            }
+            secondRowForPapalCouncils.add(button);
+        }
+        setBordersForCouncil(this.tiles, secondRowForPapalCouncils);
+        for (JButton button : this.tiles) {
+            this.add(button);
+        }
+        for (JButton button : secondRowForPapalCouncils) {
+            this.add(button);
+        }
+    }
+
+    private void setBordersForCouncil(List<JButton> firstRow, List<JButton> secondRow) {
+        for (JButton button : firstRow) {
+            if (firstRow.indexOf(button) == 4) {
+                button.setBorder(BorderFactory.createLineBorder(Color.red));
+            }
+            if (firstRow.indexOf(button) == 5 || firstRow.indexOf(button) == 6) {
+                button.setBorder(BorderFactory.createLineBorder(Color.red));
+            }
+            if (firstRow.indexOf(button) == 7) {
+                button.setBorder(BorderFactory.createLineBorder(Color.red));
+            }
+        }
+        for (JButton button : secondRow) {
+            if (button.isVisible()) {
+                button.setBorder(BorderFactory.createLineBorder(Color.red));
+            }
+        }
+    }
+
+    public static void main(String[] args) {
+        JFrame frame = new JFrame();
+        frame.setVisible(true);
+        frame.pack();
+        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        frame.add(new FaithMapGUI());
+    }
+}
